flexmeasures/data/models/charts/belief_charts.py

```diff
@@ -112,6 +112,15 @@ def chart_for_multiple_sensors(
             ]
         }
 
+    # Set up field definition for sensor descriptions
+    sensor_field_definition = FIELD_DEFINITIONS["sensor_description"]
+    sensor_field_definition["scale"] = dict(
+        domain=[
+            sensor.to_dict()["description"]
+            for sensor in flatten_unique(sensors_to_show)
```

Comment:
`flatten_unique(sensors_to_show)` is also used in *Line 95* so creating a variable could save us some *ms*.